scripts/insert_collections.md

```diff
@@ -0,0 +1,32 @@
+# Inserting Collection Metadata
+
+This document describes how to insert STAC collection metadata from a local machine into the `delta-backend-dev` database. Using cloud resources may be useful in the future for a tested and repeatable process, but a local process is all that is required at this time.
+
+These steps assumeg you can connect to the PG STAC database instance from another AWS resource (such as an EC2) or security group inbound rule permitting access for your IP.
```

Comment:
```suggestion
These steps assumes you can connect to the PG STAC database instance from another AWS resource (such as an EC2) or security group inbound rule permitting access for your IP.
```

Reply:
Good catch. I made the change to `main` since it was just a documentation change.